test/e2e/app-dir/app/app/bootstrap/page.js

```diff
@@ -0,0 +1,5 @@
+export default async function Page() {
+  return (
+    <div id="anchor">hello, bootstrap scripts should appear after this div</div>
```

Comment:
It seems like the tests are not checking for `anchor` at all.

Should they be updated to confirm that the scripts appear after this div?

Reply:
Yeah was originally going to go with that but really I don't care where react emits the bootstrap script I just care that it isn't in the head where the other resource loading scripts might be. I don't want to overly test the implementation details of react. I originally wrote this div b/c I was going to do what you suggested but I'll just update it to say hello works or something